docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,37 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::SerializableResource.serialize(post, options)
+
+# Create a json representation on your serialized model
+model_json = serializer.as_json
+```
+
+### Retrieving a Resource's Active Model Serializer
+
+If you want to retrieve a serializer for a specific resource, you can do the following:
+
+```ruby
+# Creating the resource
+post = Post.create(title: "Another Example", body: "So much fun.")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::Serializer.serializer_for(post, options)
```

Comment:
Great job here!  you could also call `ActiveModel::SerializableResource.serialize(post, options).serializer` if you already have that, though it'll either give you ArraySerializer if the resource responds to :each, else the same as ` ActiveModel::Serializer.serializer_for(post, options)`